rundeckapp/grails-app/views/execution/_wfAddStep.gsp

```diff
@@ -14,9 +14,6 @@
   - limitations under the License.
   --}%
 <%@ page import="grails.util.Environment; grails.converters.JSON" %>
-<g:if test="${grails.util.Environment.current==grails.util.Environment.DEVELOPMENT}">
-    <asset:javascript src="filterStepPluginsKOTest.js"/>
```

Comment:
Should these test files be deleted also?

Reply:
eventually it all should be migrated to another test harness, so i left them